### ISSUE TITLE
A11y_AzureDataStudioAAE_NewDeployment_AzureSQLManagedInstance_ScreenReader: NVDA does not announces "learn more" link descriptively when navigating using tab key 

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15807]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#NVDA;#WCAG2.4.4;#AINotInScope;#Jaws;

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10
NVDA Version: 2020.4, Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab to "New Deployment" and select sql managed instance and hit on select button
5. Tab to "Learn more" link start nvda and and check the announcement.

### Actual:
NVDA does not announces "learn more" link descriptively when navigating using tab key 
Issue also observed on jaws.

### Expected:
NVDA should be announcing the purpose of the "learn more" link when navigating using tab key . It should be announcing the proper information about it.

### User Impact:
Screen Reader users will not get the proper information about the learn more link

**Comment by Shivamd1** [comment]
Closing as per new guidelines of link along with text announcement.
